src/RestApi/StoreApi/Schemas/CartSchema.php

```diff
@@ -39,7 +39,16 @@ protected function get_properties() {
 					'properties' => $this->force_schema_readonly( ( new CartCouponSchema() )->get_properties() ),
 				],
 			],
-			'items'          => [
+			'selected_shipping' => [
+				'description' => __( 'List of selected shipping rates.', 'woo-gutenberg-products-block' ),
```

Comment:
List of *selected* shipping rates.

Reply:
Do you mean I need to update the `selecte_shipping` to `selected_shipping_rates`?

Reply:
Was just referring to description.